[2022-02-19 14:28:14,050] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.clean_folder manual__2022-02-19T14:27:52.738259+00:00 [queued]>
[2022-02-19 14:28:14,087] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.clean_folder manual__2022-02-19T14:27:52.738259+00:00 [queued]>
[2022-02-19 14:28:14,087] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 14:28:14,087] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 14:28:14,087] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 14:28:14,148] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): clean_folder> on 2022-02-19 14:27:52.738259+00:00
[2022-02-19 14:28:14,241] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'demo_etl', 'clean_folder', 'manual__2022-02-19T14:27:52.738259+00:00', '--job-id', '371', '--raw', '--subdir', 'DAGS_FOLDER/demo_etl.py', '--cfg-path', '/tmp/tmp7_g8hy9l', '--error-file', '/tmp/tmpr2gygggm']
[2022-02-19 14:28:14,243] {standard_task_runner.py:77} INFO - Job 371: Subtask clean_folder
[2022-02-19 14:28:14,270] {standard_task_runner.py:52} INFO - Started process 15933 to run task
[2022-02-19 14:28:14,470] {logging_mixin.py:109} INFO - Running <TaskInstance: demo_etl.clean_folder manual__2022-02-19T14:27:52.738259+00:00 [running]> on host d18a6d5a39bc
[2022-02-19 14:28:14,649] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=demo_etl
AIRFLOW_CTX_TASK_ID=clean_folder
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T14:27:52.738259+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-19T14:27:52.738259+00:00
[2022-02-19 14:28:14,650] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-19 14:28:14,662] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'find . -name "*.csv" -type f']
[2022-02-19 14:28:14,799] {subprocess.py:85} INFO - Output:
[2022-02-19 14:28:15,302] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-19 14:28:15,444] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=demo_etl, task_id=clean_folder, execution_date=20220219T142752, start_date=20220219T142814, end_date=20220219T142815
[2022-02-19 14:28:15,513] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 14:28:15,600] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
